docs/source/code-generation/codegen-getting-started.mdx

```diff
@@ -60,24 +54,17 @@ query Query {
 }
 ```
 
-## Step 4: Compile the Codegen CLI
-
-In the root directory of the `apollo-ios` project there is a `makefile` which includes a `build-cli` rule that can be used to compile the CLI. From the root of your `apollo-ios` clone, in a Terminal run:
-
-```bash
-make build-cli
-```
-
-This will compile the Codegen CLI in release mode for you. The Codegen CLI executable will now be located in your `apollo-ios` clone at `.build/release/apollo-ios-cli`.
+## Step 5: Download the CLI
 
+Browse to the list of [Apollo iOS releases](https://github.com/apollographql/apollo-ios/releases) in GitHub and find the latest release. In the Assets list for each release will be a pre-built CLI binary.
 
-## Step 5: Move the CLI executable
+Download or move the already downloaded CLI binary file into the directory you created in [Step 1](#step-1-setup).
```

Comment:
Just to confirm on this step @calvincestari 

Will apollo-ios-cli.tar.gz be available as an asset on every release?

Also don’t they have to unzip this file to use it?

Reply:
I pushed up a change to fix this.

Reply:
> Will apollo-ios-cli.tar.gz be available as an asset on every release?

Yes

> Also don’t they have to unzip this file to use it?

Yes